wrappers/python/nntile/tensor.py

```diff
@@ -11,811 +11,6 @@
-        raise TypeError
-
-# Wrapper for multiprecision hypot
-def hypot_async(alpha: float, x: Tensor, beta: float, y: Tensor) -> None:
-    if type(x) is not type(y):
-        raise TypeError
-    if type(x) is core_tensor.Tensor_fp32:
-        core_tensor.hypot_async_fp32(alpha, x, beta, y)
-    elif type(x) is core_tensor.Tensor_fp64:
-        core_tensor.hypot_async_fp64(alpha, x, beta, y)
-    else:
-        raise TypeError
-
-# Wrapper for multiprecision hypot_scalar_inverse
-def hypot_scalar_inverse_async(eps: float, alpha: float, x: Tensor) -> None:
-    if type(x) is core_tensor.Tensor_fp32:
-        core_tensor.hypot_scalar_inverse_async_fp32(eps, alpha, x)
-    elif type(x) is core_tensor.Tensor_fp64:
-        core_tensor.hypot_scalar_inverse_async_fp64(eps, alpha, x)
-    else:
-        raise TypeError
-
-def fused_adam_step(p: Tensor, grad: Tensor, first_moment: Tensor, second_moment: Tensor,
-                   lr: float, eps: float, beta1: float, beta2: float, weight_decay: float, num_iter: int):
-    if type(p) is not type(grad):
-        raise TypeError
-    if type(p) is not type(first_moment):
-        raise TypeError
-    if type(p) is not type(second_moment):
-        raise TypeError
-    if type(p) is core_tensor.Tensor_fp32:
-        core_tensor.adam_step_async_fp32(num_iter, beta1, beta2, eps, lr, weight_decay,
-                                         grad, first_moment, second_moment, p)
-    elif type(p) is core_tensor.Tensor_fp64:
-        core_tensor.adam_step_async_fp64(num_iter, beta1, beta2, eps, lr, weight_decay,
-                                         grad, first_moment, second_moment, p)
-    else:
-        raise TypeError
-
-def fused_adamw_step(p: Tensor, grad: Tensor, first_moment: Tensor, second_moment: Tensor,
-                   lr: float, eps: float, beta1: float, beta2: float, weight_decay: float, num_iter: int):
-    if type(p) is not type(grad):
-        raise TypeError
-    if type(p) is not type(first_moment):
-        raise TypeError
-    if type(p) is not type(second_moment):
-        raise TypeError
-    if type(p) is core_tensor.Tensor_fp32:
-        core_tensor.adamw_step_async_fp32(num_iter, beta1, beta2, eps, lr, weight_decay,
-                                         grad, first_moment, second_moment, p)
-    elif type(p) is core_tensor.Tensor_fp64:
-        core_tensor.adamw_step_async_fp64(num_iter, beta1, beta2, eps, lr, weight_decay,
-                                         grad, first_moment, second_moment, p)
-    else:
-        raise TypeError
-
-# Wrapper for multiprecision transpose
-def transpose_async(alpha: float, src: Tensor, dst: Tensor, ndim: int) -> None:
-    if type(src) is not type(dst):
-        raise TypeError
-    if type(src) is core_tensor.Tensor_fp32:
-        core_tensor.transpose_async_fp32(alpha, src, dst, ndim)
-    elif type(src) is core_tensor.Tensor_fp64:
-        core_tensor.transpose_async_fp64(alpha, src, dst, ndim)
-    else:
-        raise TypeError
```

Comment:
Do not rework anything in feature PR. Prepend this PR with a PR where you rework this module.

Reply:
moved here: https://github.com/nntile/nntile/pull/57
I will rebase this later